schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/CommonInterpreted.avdl

```diff
@@ -248,101 +301,81 @@ It is a representation of the zygosity
         string reference;
 
         /**
-        alternate: Alternate Allele sequence, the same provided in vcf
+        alternate: Alternate Allele sequence, the same provided in vcf. Only one
+        allele is supported
         */
         string alternate;
 
-        /**
-        array of genotypes for the family
-        */
-        array<CalledGenotype> calledGenotypes;
-
-        /**
-        This is the scores across multiple modes of inheritance, for each model of inheritance analyzed,
-        the variants can have only one Report event.
-        */
-        array<ReportEvent> reportEvents;
+        Assembly assembly;
 
-        /**
-        For example a quote from a paper
-        */
-        union {null,map<string>} additionalTextualVariantAnnotations;
+    }
 
-        /**
-        For example HGMD ID, dbSNP ID or Pubmed Id
-        */
-        union {null,map<string>} evidenceIds;
+    record VariantAttributes {
+        union {null, int} ihp;
+        union {null, boolean} recurrentlyReported;
+        union {null, string} fdp50;
+        union {null, map<string>} others;
+    }
 
+    record ReportedVariant {
 
+        VariantCoordinates variantCoordinates;
 
         /**
-        For Example (Allele Frequency, sift, polyphen, mutationTaster, CADD. ..)
+        variant ID in dbSNP
         */
-        union {null, map<float>} additionalNumericVariantAnnotations;
+        union {null, string} dbSnpId;
 
         /**
-        Comments
+        variant ID in Cosmic
         */
-        union {null, array<string>} comments;
-
-    }
-
-
-
-    record ReportedStructuralVariant {
+        union {null, array<string>} cosmicIds;
 
         /**
-        chromosome, named as: 1-22,X,Y,MT(other contigs name)
+        variant ID in ClinVar
         */
-        string chromosome;
+        union {null, array<string>} clinVarIds;
 
         /**
-        start: start position 1-based
+        gDNA change, HGVS nomenclature (e.g.)
         */
-        int start;
+        union {null, array<string>} genomicChange;
 
         /**
-        end: end position 1-based
+        cDNA change, HGVS nomenclature (e.g.)
         */
-        int end;
+        union {null, array<string>} cdnaChanges;
 
         /**
-        The ID field indicates the type of structural variant, and can be a colon-separated list of types and subtypes
-        (this is VCF Format): DEL, INS, DUP, INV, CNV, DUP:TANDEM, DEL:ME, INS:ME, INS:ME:ALU...
+        protein change, HGVS nomenclature (e.g.)
         */
-        string type;
+        union {null, array<string>} proteinChanges;
 
-        /**
-        reference: Reference Allele sequence, the same provided in vcf
-        */
-        string reference;
 
         /**
-        alternate: Alternate Allele sequence, the same provided in vcf
+        array of genotypes for the samples
         */
-        string alternate;
-
-
-        array<CalledGenotype> calledGenotypes;
+        array<VariantCall> variantCalls;
```

Comment:
If VariantCalls is in CommonInterpreted, is there a "na" for somatic variants?

Reply:
Nop,  VariantCalls is still valid for somatic variants zygosity (inside VariantCalls) is the one that will be na